msteams-platform/toolkit/TeamsFx-multi-env.md

```diff
@@ -8,212 +8,50 @@ ms.topic: conceptual
 ms.date: 11/29/2021
 ---
 
-# Manage multiple environments
+# Environments in Teams Toolkit
 
- Teams Toolkit provides a simple way for you to create and manage multiple environments. You can use environments to provision resources and deploy artifacts to the target environment for your Microsoft Teams app.
+Teams Toolkit provides a way to run and test your app in different targets for deployment like dev, staging, production, or locally on your machine. To do this, Teams Toolkit includes an optional set of features that orchestrate the configuration of these different targets using information in environment files. The environment files represent a collection of all the resources of a deployment target. Teams Toolkit project templates include two default environment groupings - one for running the app on your machine named "local" and another for hosting somewhere else named "dev". You can rename the "dev" environment and add additional environments using the Environments window or manually with a new `.env.{environment-name}` file. You can't rename the local environment.
 
- You can perform the following activities with environments:
+By supplying different environment files, Teams Toolkit can help automate the provisioning of resources needed to run the app and deploy your code to those resources. Using these features means you can have a single set of deployment rules for any number of environments.
 
-1. **Test before production**: You can set up multiple environments, such as dev, test, and staging before publishing a Teams app to production environment in modern app development lifecycle.
+## Using environment files with Teams Toolkit
 
-2. **Manage app behaviors in different environments**: You can set up different app behaviors for multiple environments, such as enabling telemetry in the production environment.
+Teams Toolkit uses a naming convention for the environment files like: `.env.{environment-name}`. When using the toolkit's lifecycle commands (Provision, Deploy, Publish) you're required to provide an environment name. The name of the environment should be the same as the value chosen for`{environment-name}` when creating the environment file.
 
-   > [!NOTE]
-   > Ensure that telemetry is disabled in the development environment.
+> [!NOTE]
+> You can also define environment variables for Teams Toolkit in your current shell environment which override any values in a `.env` file during a naming conflict.
 
-## Create a new environment
+### Environment file location
 
-Each project can have one **local** environment but multiple remote environments. After you create a project, Teams Toolkit configures the following default environments:
+You can define where `.env` files are using the `environmentFolderPath` property in `teamsapp.yml` and `teamsapp.local.yml`. By default, Teams Toolkit assumes these files are an `./env/` directory.
```

Comment:
`environmentFolderPath` is only read from `teamsapp.yml` not from `teamsapp.local.yml`.

Reply:
Updated.